data/human/human missions.txt

```diff
@@ -3996,3 +3996,89 @@ mission "Fumbling Mugger"
+			`	"I appreciate your conundrum," replies the woman, her voice clipped. "However, my policy is cash upfront. Good day."`
+			`	She turns and walks away briskly, and the man slumps against the bar.`
+			choice
+				`	"What was that all about?"`
+				`	(Leave before he notices you).`
+					decline
+			`	He gives you a once-over, and you see a spark of hope in his eyes. "You're a captain? Please, I need to get to <destination>. Work has dried up here, and I need a fresh start."`
+			choice
+				`	"And what's your offer?"`
+					goto offer
+				`	"Work dried up on the whole planet?"`
+			`	"This planet is messed up. Everybody likes to talk behind your back. I did a job for one of the big mining companies here, but my boss had it out for me. Told everybody I did it wrong, even though they never gave me any training at all. Then he went and put me on the permanent record. So yeah, sooner I get off this dump, the better."`
+			label offer
+			`	His eyes drop. "I can pay you after I earn the money. I've just gotta get to <planet> to get work, and then I'll send the money. I just want to earn a living, same as anyone."`
+			choice
+				`	"Okay, I can take you to <planet>."`
+					goto accept
+				`	"How do I know you'll pay?"`
+					goto question
+				`	"Sorry, money up front."`
+					goto decline
+			label question
+				`	"You don't trust me? Just keep walking, then, if you're gonna be like that."`
+			choice
+				`	"Sorry, money up front."`
+					goto decline
+				`	"Okay, I can take you to <planet>."`
+			label accept
+			`	"I'll be on board. There's no rush, honestly. I need a break from all this stress."`
+				accept
+			label decline
+				`	"I knew it. You're just like everyone else."`
+					decline
+	on complete
+		conversation
+			`The worker you picked up from <source> spent every possible moment of the voyage telling his story. He's had a tough life, full of unlucky setbacks, unfair treatment, and outright persecution from others. You think that you'd feel more sorry for him if you weren't so eager to get his ceaseless negativity off your ship.`
+			`	He carries his bags to the lowered ramp, then turns back before disembarking.`
+			`	"Thanks for the lift. You're one of the good ones, you know?"`
+			choice
+				`	"You're welcome. Better luck on this world."`
+					goto luck
+				`	"You have my bank address, right?"`
+					goto bank
+				`	"I need that money now."`
+			`	He shoots you an accused look. "You said I could pay you later. As we discussed, I'll send you the payment when I have it."`
+				goto end
+			label bank
+				`	He shoots you a look of thinly masked annoyance. "Yeah. I'll send you the credits when I have them. Bye, now."`
+					goto end
+			label luck
+				`	He bobs his head at you in farewell. "I'll send you the payment when I have it."`
+			label end
+				`	He turns and walks off.`
+
+event "laborer payment"
+
+mission "Laborer Payment"
+	landing
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Quarg" "Neutral"
```

Comment:
Not including "Independent" planets? Also make sure this mission doesn't offer on alien Quarg worlds.